compose-builder/docker-compose-base.yml

```diff
@@ -27,21 +28,38 @@ volumes:
   kuiper-plugins:
 
 services:
-  database:
-    image: redis:${REDIS_VERSION}
-    user: "root:root" # Note that Redis is run under the 'redis' user, but entry point scripts need to first run as root
+  keeper:
```

Comment:
could you please rename it to core-keeper?

Reply:
Updated.